{"ResponseBody":{"UserMessage":"Fail to update leave deatil. Please contact to admin","InnerMessage":null,"StackTrace":"   at ServiceLayer.Code.LeaveCalculation.UpdateEmployeesRecord(List`1 employeeAccrualData) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 247\r\n   at ServiceLayer.Code.LeaveCalculation.RunAccrualCycleByEmployee(Int64 EmployeeId) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 212\r\n   at ServiceLayer.Code.LeaveService.RunAccrualByEmployeeService(Int64 EmployeeId) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveService.cs:line 450\r\n   at OnlineDataBuilder.Controllers.LeaveController.RunAccrualByEmployee(Int64 EmployeeId) in E:\\Marghub\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\LeaveController.cs:line 124\r\n   at lambda_method(Closure , Object )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in E:\\Marghub\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in E:\\Marghub\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":400,"HttpStatusMessage":"Fail to update leave deatil. Please contact to admin","AuthenticationToken":""}{"ResponseBody":{"UserMessage":"From date: 3/22/2023 12:00:00 AM already exist in another leave request","InnerMessage":null,"StackTrace":"   at ServiceLayer.Code.LeaveCalculation.CheckSameDateAlreadyApplied(List`1 completeLeaveDetails, LeaveCalculationModal leaveCalculationModal) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 474\r\n   at ServiceLayer.Code.LeaveCalculation.SameDayRequestValidationCheck(LeaveCalculationModal leaveCalculationModal) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 485\r\n   at ServiceLayer.Code.LeaveCalculation.PrepareCheckLeaveCriteria(LeaveRequestModal leaveRequestModal) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 326\r\n   at ServiceLayer.Code.LeaveCalculation.CheckAndApplyForLeave(LeaveRequestModal leaveRequestModal) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 613\r\n   at ServiceLayer.Code.LeaveService.ApplyLeaveService(LeaveRequestModal leaveRequestModal, IFormFileCollection FileCollection, List`1 fileDetail) in E:\\Marghub\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveService.cs:line 413\r\n   at OnlineDataBuilder.Controllers.LeaveController.ApplyLeave() in E:\\Marghub\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\LeaveController.cs:line 108\r\n   at lambda_method(Closure , Object )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in E:\\Marghub\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in E:\\Marghub\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":400,"HttpStatusMessage":"From date: 3/22/2023 12:00:00 AM already exist in another leave request","AuthenticationToken":""}{"ResponseBody":{"UserMessage":"Leave detail not found for employee id: 4","InnerMessage":null,"StackTrace":"   at ServiceLayer.Code.LeaveCalculation.GetLeaveDetailService(Int64 EmployeeId) in D:\\ws\\Server\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveCalculation.cs:line 103\r\n   at ServiceLayer.Code.LeaveService.GetLatestLeaveDetail(Int64 employeeId) in D:\\ws\\Server\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveService.cs:line 438\r\n   at ServiceLayer.Code.LeaveService.GetEmployeeLeaveDetail(LeaveRequestModal leaveRequestModal) in D:\\ws\\Server\\OnlineDataBuilderServer\\ServiceLayer\\Code\\LeaveService.cs:line 448\r\n   at OnlineDataBuilder.Controllers.LeaveController.GetAllLeavesByEmpId(LeaveRequestModal leaveRequestModal) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\LeaveController.cs:line 117\r\n   at lambda_method(Closure , Object )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":400,"HttpStatusMessage":"Leave detail not found for employee id: 4","AuthenticationToken":""}{"ResponseBody":{"UserMessage":"Fail to insert or update leave plan detail.","InnerMessage":null,"StackTrace":"   at ServiceLayer.Code.ManageLeavePlanService.UpdateLeaveAccrualService(Int32 leavePlanTypeId, Int32 leavePlanId, LeaveAccrual leaveAccrual) in D:\\ws\\Server\\OnlineDataBuilderServer\\ServiceLayer\\Code\\ManageLeavePlanService.cs:line 200\r\n   at OnlineDataBuilder.Controllers.ManageLeavePlanController.UpdateLeaveAccrual(Int32 leavePlanTypeId, Int32 leavePlanId, LeaveAccrual leaveAccrual) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\ManageLeavePlanController.cs:line 47\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":400,"HttpStatusMessage":"Fail to insert or update leave plan detail.","AuthenticationToken":""}{"ResponseBody":{"UserMessage":"Fail to insert or update leave plan detail.","InnerMessage":null,"StackTrace":"   at ServiceLayer.Code.ManageLeavePlanService.UpdateLeaveAccrualService(Int32 leavePlanTypeId, Int32 leavePlanId, LeaveAccrual leaveAccrual) in D:\\ws\\Server\\OnlineDataBuilderServer\\ServiceLayer\\Code\\ManageLeavePlanService.cs:line 200\r\n   at OnlineDataBuilder.Controllers.ManageLeavePlanController.UpdateLeaveAccrual(Int32 leavePlanTypeId, Int32 leavePlanId, LeaveAccrual leaveAccrual) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\ManageLeavePlanController.cs:line 47\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":400,"HttpStatusMessage":"Fail to insert or update leave plan detail.","AuthenticationToken":""}{"ResponseBody":{"UserMessage":"Fail to insert or update leave plan detail.","InnerMessage":null,"StackTrace":"   at ServiceLayer.Code.ManageLeavePlanService.UpdateLeaveAccrualService(Int32 leavePlanTypeId, Int32 leavePlanId, LeaveAccrual leaveAccrual) in D:\\ws\\Server\\OnlineDataBuilderServer\\ServiceLayer\\Code\\ManageLeavePlanService.cs:line 200\r\n   at OnlineDataBuilder.Controllers.ManageLeavePlanController.UpdateLeaveAccrual(Int32 leavePlanTypeId, Int32 leavePlanId, LeaveAccrual leaveAccrual) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\Controllers\\ManageLeavePlanController.cs:line 47\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at SchoolInMindServer.MiddlewareServices.RequestMiddleware.Invoke(HttpContext context, CurrentSession currentSession) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\RequestMiddleware.cs:line 65\r\n   at SchoolInMindServer.MiddlewareServices.ExceptionHandlerMiddleware.Invoke(HttpContext context, ApplicationConfiguration applicationConfiguration) in D:\\ws\\Server\\OnlineDataBuilderServer\\OnlineDataBuilder\\MiddlewareServices\\ExceptionHandlerMiddleware.cs:line 28"},"HttpStatusCode":400,"HttpStatusMessage":"Fail to insert or update leave plan detail.","AuthenticationToken":""}